tools/UpdatePaths/Scripts/27268_snake_case_remap.txt

```diff
@@ -22,7 +22,7 @@
 /mob/living/simple_animal/pet/dog/fox/Syndifox/@SUBTYPES : /mob/living/simple_animal/pet/dog/fox/syndifox/@SUBTYPES{@OLD}
 /mob/living/simple_animal/hostile/bear/Hudson/@SUBTYPES : /mob/living/simple_animal/hostile/bear/hudson/@SUBTYPES{@OLD}
 /mob/living/simple_animal/pet/dog/fox/Renault/@SUBTYPES : /mob/living/simple_animal/pet/dog/fox/renault/@SUBTYPES{@OLD}
-/obj/item/gun/projectile/automatic/pistol/APS/@SUBTYPES : /obj/item/gun/projectile/automatic/pistol/aps/@SUBTYPES{@OLD}
+/obj/item/gun/projectile/automatic/pistol/type_230/@SUBTYPES : /obj/item/gun/projectile/automatic/pistol/type_230/@SUBTYPES{@OLD}
```

Comment:
this and the file below should not be modified. this is historical data for a previous migration.